src/patterns/start-pages/index.md.njk

```diff
@@ -32,19 +36,9 @@ Start pages include 4 main elements:
 
 This is a set pattern, so you won’t be able to customise it. 
 
-{{ example({group: 'patterns', item: 'start-pages', example: 'default', html: true, nunjucks: true, open: true}) }}
-
-### Testing start pages in discovery and alpha
-
-If you want to prototype designs so you can test them with users, you can use the [start page template](https://govuk-prototype-kit-beta.herokuapp.com/docs/examples/start-page) in the [GOV.UK Prototype Kit](https://govuk-prototype-kit-beta.herokuapp.com/).
+Read guidance on [what information should go on a start page](www.gov.uk/service-manual/design/govuk-content-transactions) in the Service Manual.
```

Comment:
This needs `https://` at the start of the URL – at the minute it would link to `https://design-system.service.gov.uk/patterns/start-pages/www.gov.uk/service-manual/design/govuk-content-transactions`.

(This is why the build has failed – because the broken link test failed)